### DISCUSSION TITLE
Send message current user

### DISCUSSION THREAD

**Discussion by root-aza** [centrifugal/centrifugo#605] [Q&A] [answered]
Generated jwt:

<details>
<img width="973" alt="Screenshot 2023-01-12 at 18 06 13" src="https://user-images.githubusercontent.com/13816704/212050780-414a40e3-44fb-478b-ba81-8a9c241e6aad.png">
</details>

Call publish:

<details>
<img width="1411" alt="Screenshot 2023-01-12 at 18 06 50" src="https://user-images.githubusercontent.com/13816704/212050898-981d4832-0e8e-41b8-9f8c-aafc038f895c.png">
</details>

Why the message to the specific user does not come? If we send to the entire channel, then everything is ok.

config.json

<details>

```json
{
    "v3_use_offset": true,
    "token_hmac_secret_key": "CPQFZIBI4vN0ha141",
    "api_key": "wDQHvgfWGye5qUvTkRMCxJra94Xp7FRZhwJYEtLz",
    "admin_password": "password",
    "admin_secret": "secret",
    "admin": true,
    "allowed_origins": [
        "*"
    ],
    "publish": true,
    "proxy_publish": true,
    "proxy_subscribe": true,
    "proxy_connect": true,
    "allow_subscribe_for_client": true,
    "grpc_api": true,
    "grpc_api_address": "0.0.0.0",
    "grpc_api_port": 10000,
    "proxy_connect_endpoint": "grpc://docker.for.mac.host.internal:10001",
    "proxy_connect_timeout": "10s",
    "proxy_publish_endpoint": "grpc://docker.for.mac.host.internal:10001",
    "proxy_publish_timeout": "10s",
    "proxy_subscribe_endpoint": "grpc://docker.for.mac.host.internal:10001",
    "proxy_subscribe_timeout": "10s",
    "proxy_refresh_endpoint": "grpc://docker.for.mac.host.internal:10001",
    "proxy_refresh_timeout": "10s",
    "proxy_rpc_endpoint": "grpc://docker.for.mac.host.internal:10001",
    "proxy_rpc_timeout": "10s",
    "uni_sse": true,
    "namespaces": [
        {
            "name": "auth",
            "allow_subscribe_for_client": true
        }
    ]
}

```

</details>

**Comment by FZambia** [comment, marked as answer]
Hello, from what I see, you subscribe user to channel `auth:qr`, but publishing to `auth:qr#150,250`. See https://centrifugal.dev/docs/server/channels#channel-is-just-a-string docs, it should give the idea how things work in Centrifugo. TLDR you need to publish to `auth:qr` for user to receive the message in this case.
